.github/workflows/static_check.yml

```diff
@@ -0,0 +1,25 @@
+name: Static Check
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  static_check:
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Prepare Environment
+      run: |
+        sudo apt-get update && \
+        sudo apt-get install -y dpkg-dev debhelper g++ libncurses6 pkg-config \
```

Comment:
Do we really need `debhelper` for static checks?

Reply:
neither dpkg-dev probably, will adjust/minimize